package.json

```diff
@@ -1,45 +0,0 @@
-{
```

Comment:
If this is now a link, why is the root package.json linking to the one in eve directory, shouldn't it be the opposite way?

Reply:
Apparently the `npm install` step in `Dockerfile` could not use symlinks. So @rayene did it the other way around.

Reply:
We have to check, how npm install works though.

Reply:
Well it's not a problem with npm, but rather with docker.
Docker doesn't suppert symlink. I think it is a safety to avoid being able to point to file outside of the docker context (the directory where the docker build is run).